moto/mq/responses.py

```diff
@@ -89,32 +89,32 @@ def update_broker(self) -> str:
         )
         return self.describe_broker()
 
-    def delete_broker(self) -> str:
+    def delete_broker(self) -> ActionResult:
         broker_id = self.path.split("/")[-1]
         self.mq_backend.delete_broker(broker_id=broker_id)
-        return json.dumps(dict(brokerId=broker_id))
+        return ActionResult(dict(BrokerId=broker_id))
 
-    def describe_broker(self) -> str:
+    def describe_broker(self) -> ActionResult:
         broker_id = self.path.split("/")[-1]
         broker = self.mq_backend.describe_broker(broker_id=broker_id)
-        resp = broker.to_json()
-        resp["tags"] = self.mq_backend.list_tags(broker.arn)
-        return json.dumps(resp)
+        resp = copy.copy(broker)
+        setattr(resp, "tags", self.mq_backend.list_tags(broker.arn))
+        return ActionResult(resp)
```

Comment:
Generally speaking, backend models should try to align with response models as closely as possible, but the response methods will still have the ability to pull information from disparate sources (multiple backend methods in this case) and combine them into one result.  This particular example is a little hacky, but works with the existing code.  Alternatively, a `tags` attribute could be added to the `broker` backend model.